test/kubemark/stop-kubemark.sh

```diff
@@ -45,22 +48,27 @@ rm -rf "${RESOURCE_DIRECTORY}/addons" \
     "${RESOURCE_DIRECTORY}/hollow-node.yaml"  &> /dev/null || true
 
 if [[ "${SCALEOUT_CLUSTER:-false}" == "true" ]]; then
-  export KUBE_ENABLE_APISERVER_INSECURE_PORT=true
+  export KUBE_ENABLE_APISERVER_INSECURE_PORT=${ENABLE_APISERVER_INSECURE_PORT}
```

Comment:
With above KUBE_ENABLE_APISERVER_INSECURE_PORT, this is not needed

Reply:
outdated comment. ENABLE_APISERVER_INSECURE_PORT is removed